### DISCUSSION TITLE
Gamecube Wiring

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#48] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17175988/123416561-a12c6880-d5ae-11eb-9058-0d8890c7c30d.png)

The above shows that there should be something connecting to the shield but I don't seem to have a wire connecting there - is is safe to assume it's already linked to the ground inside the connector?

**Comment by darthcloud** [comment, marked as answer]
GND is GND, if you got at least one its fine.

**Reply by vaderag** [reply]
One additional question - connection 6 looks like it's 3.3V on the gamecube controller but doesnt need connecting - can you foresee a time that it might come in useful/needed or best just to snip and terminate?

**Reply by darthcloud** [reply]
Not needed ever.
